fluidBuild.config.cjs

```diff
@@ -388,7 +388,6 @@ module.exports = {
 				// Packages that violate the API linting rules
 				// ae-missing-release-tags, ae-incompatible-release-tags
 				"^examples/data-objects/table-document/",
-				// AB#8147: ./test/EditLog export should be ./internal/... or tagged for support
```

Comment:
You are removing the comment about why this package is being special cases in this file, but not the fact that it is special cased.